applications/ConvectionDiffusionApplication/python_scripts/convection_diffusion_solver.py

```diff
@@ -79,7 +79,6 @@ def GetDefaultParameters(cls):
             "domain_size" : -1,
             "echo_level": 0,
             "analysis_type": "linear",
-            "time_integration_method": "implicit",
```

Comment:
100% agree, this cannot be here if it is only needed in some of the derived solvers